src/dfx/assets/new_project_base_files/dfx.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": 1,
+    "canisters": {},
+    "defaults": {
+        "build": {
+            "packtool": "",
+            "args": ""
+        }
+    },
```

Comment:
```suggestion
```
These lines serve no purpose AFAIK. Is it worth keeping them?

Reply:
If it should be removed, I'll remove it when I update the templates themselves.